controllers/controllers_suite_test.go

```diff
@@ -183,6 +180,14 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
+	projectDir := os.Getenv("PROJECT_DIR")
+	// Add ginkgo recover statements to controllers.
+	cmd := exec.Command(projectDir+"/hack/testing_ginkgo_recover_statements.sh", "--remove")
+	cmd.Stdout = os.Stdout
+	if err := cmd.Run(); err != nil {
+		fmt.Println("Refusing to run tests without ginkgo recover set.")
```

Comment:
this error message doesn't make sense for cleanup

Reply:
Good point, thanks!